tests/api/test_loans_rest.py

```diff
@@ -198,8 +198,8 @@ def test_overdue_loans(client, librarian_martigny_no_email,
     patron_pid = patron_martigny_no_email.pid
     assert not item.is_loaned_to_patron(patron_martigny_no_email.get(
         'barcode'))
-    assert item.can_delete
-    assert item.available
+    assert not item.can_delete
```

Comment:
any explanation for this? Was it a bug?

Reply:
Probably is a fixture problem, I removed it because this test already done in the same file in an earlier function. I also created a task to ensure that `item.can_delete` considers only active loans.